src/daily_output.ts

```diff
@@ -120,7 +120,10 @@ export class DailyOutput {
 			return volumeDiff || tipsDiff || blockSinceParticipatingDiff;
 		});
 
-		shuffledTies.forEach((address, index) => (this.data.wallets[address].daily.rankPosition = index + 1));
```

Comment:
Not necessarily from this PR, but `finishDataAggregation` has no async calls. It doesn't need to be a promise, as well as its "parent" method `feedGQLData`

Reply:
50488de6b01df8d115cda387d69d1c907b3f646b